buddyboss/buddyboss-platform
# Fix - Unable to post a forum if page has a pop up from elementor

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3106 
Fixes #3076

### How to test the changes in this Pull Request:

1. Create new elementor popup with page onload trigger add it for entire website
2. Go to any forum discussion
3. Create new discussion with dummy content, It should work without any error
4. Create new reply in any discussion with dummy content, It should work without any error

### Proof Screenshots or Video
https://www.loom.com/share/b44d6f9b68cb4c9f8ae160c7714956d6

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
